### DISCUSSION TITLE
Restore from an basebackup

### DISCUSSION THREAD

**Discussion by MrGithub89** [docker-library/postgres#1200] [General]
Hello,

my colleges and myself are struggeling to recover an basebackup (tar.gz) in your Postgresql Pod in our Kubernetes Cluster. We tried very much all "how-to" Instructions that we could find in the WWW. But without any success.

Is there any kind of Readme or Instruction on how to do that?

Many thanks for the help 😊

**Comment by tianon** [comment]
I'be found a lot of info in https://www.postgresql.org/docs/current/app-pgbasebackup.html and https://www.postgresql.org/docs/current/backup.html about how to create a basebackup, but it's not very clear on how to restore one. :sweat_smile:

Maybe https://stackoverflow.com/a/50876452/433558 and/or the page it links to (https://web.archive.org/web/20210624212358/http://postgresql.freeideas.cz/pg_basebackup-pgbarman-restore-tar-backup/) are useful?
